src/str/utf8/introspect.php

```diff
@@ -0,0 +1,112 @@
+<?hh // strict
+/*
+ *  Copyright (c) 2004-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+namespace HH\Lib\Str\Utf8;
+
+use namespace HH\Lib\_Private;
+use type HH\Lib\Str\Encoding;
+
+/**
+ * Returns the length of the given string.
+ * A multi-byte character is counted as 1.
+ *
+ * Previously known in PHP as `mb_strlen`.
```

Comment:
nit: Consistent between "Previously known in PHP as *" vs. "Previously known as * in PHP". We've gotten this wrong too, lol.